spec/controllers/notes_controller_spec.rb

```diff
@@ -109,50 +109,53 @@
     end
 
     context 'POST on :create' do
-      it 'be able to create with empty note' do
-        post_as @ta,
-                :create,
-                params: { noteable_type: 'Grouping', note: { noteable_id: @grouping.id } }
-        expect(assigns :note).not_to be_nil
-        expect(flash.empty?).to be_truthy
-        expect(assigns :assignments).not_to be_nil
-        expect(assigns :groupings).not_to be_nil
-      end
+      describe 'When grader is trying to create notes' do
+        it 'be able to create with empty note' do
+          post_as @ta,
+                  :create,
+                  params: { noteable_type: 'Grouping', note: { noteable_id: @grouping.id } }
+          expect((assigns :note)).not_to be_nil
```

Comment:
just a general comment: 

`assigns(:note)` is preferable to `(assigns :note)`. I'm actually surprised houndbot doesn't complain about this one.